app/pages/covid19/ui.test.js

```diff
@@ -0,0 +1,79 @@
+import { Selector, ClientFunction } from 'testcafe';
+import { extractInnerText } from '../../test-utils/helper';
+
+import manifest from './manifest';
+import covid19Data from './data/covid19-solutions';
+
+const pageSetup = async (t) => {
+  await t.navigateTo('http://localhost:1234/solutions/covid19');
+};
+
+fixture('Show covid19 Page');
+
+test('should display the covid19 page title', async (t) => {
+  await pageSetup(t);
+  const pageTitle = Selector('h1[data-test-id="general-page-title"]');
+  await t
+    .expect(pageTitle.exists).ok()
+    .expect(await extractInnerText(pageTitle)).eql(manifest.title);
+});
+
+test('should display the covid page description', async (t) => {
+  await pageSetup(t);
+  const pageDescription = Selector('h2[data-test-id="general-page-description"]');
+  await t
+    .expect(pageDescription.exists).ok()
+    .expect(await extractInnerText(pageDescription)).eql(manifest.description);
+});
+
+test('should display the covid19 solution cards', async (t) => {
+  await pageSetup(t);
+  await t
+    .expect(Selector('div[data-test-id="solution-card-covid19"]').count).eql(4);
```

Comment:
try and not hardcode the `4` here. Do it based on the size of the solutions in data